README.md

```diff
@@ -30,131 +30,69 @@
 
 </div>
 
-# Beta9
+### Beta9 is an open source platform for running multi-cloud GPU workloads
```

Comment:
How about something like

`Beta9 is an open source container orchestrator optimized for running GPU workloads across regions and clouds.`